docker/diff/diff.py

```diff
@@ -0,0 +1,197 @@
+from google.cloud import bigquery
+from google.cloud.exceptions import GoogleCloudError
+from typing import Dict
+from enum import Enum
+import os
+
+# "broad-datarepo-terra-prod-cgen"
+repo_data_project = os.getenv("REPO_DATA_PROJECT")
+# "datarepo_broad_dsp_clinvar"
+repo_dataset_name = os.getenv("REPO_DATASET_NAME")
+# release_date = "2020-06-02"
+release_date = os.getenv("RELEASE_DATE")
+# prev_date = "2020-05-06"
+prev_date = os.getenv("PREV_DATE")
+# "broad-dsp-monster-clingen-prod"
+dest_data_project = os.getenv("DEST_DATA_PROJECT")
+# dest_dataset_name = "raaidTest1"
+dest_dataset_name = os.getenv("DEST_DATASET_NAME")
+# dest_bucket_name = "broad-dsp-monster-clingen-dev-ingest-results"
+dest_bucket_name = os.getenv("DEST_BUCKET_NAME")
+# path of table to extract to
+dest_path = os.getenv("DEST_PATH")
+# the column name with the date/time, so "release_date" for clinvar
+version_col = os.getenv("VERSION_COL_NAME")
+# CREATE, DELETE, or UPDATE
+diff_type = os.getenv("DIFF_TYPE")
+# table name that should exist in the dataset
+table_name = os.getenv("TABLE_NAME")
+# make sure to pass in GOOGLE_APPLICATION_CREDENTIALS
+
+repo_client = bigquery.Client(project=repo_data_project) if repo_data_project else None
+dest_client = bigquery.Client(project=dest_data_project) if dest_data_project else None
+
+
+class DiffType(Enum):
+    CREATE = "created"
+    DELETE = "deleted"
+    UPDATE = "updated"
+
+
+def create_dataset(dataset_name: str = dest_dataset_name, client: bigquery.Client = dest_client) -> str:
```

Comment:
This doesn't appear to be used.

Reply:
Correct; I'd like to have this function or something similar be the way we create datasets moving forward (I thought I'd need to create one but I can piggyback off of a previous step's dataset creation). I can remove it and store it somewhere else for now if that makes more sense.

Reply:
OK to leave it here, I just wanted to make sure I was reading things right.